docs/how_it_works.rst

```diff
@@ -47,18 +47,16 @@ these as *passive* and *active* discovery.
   This is enabled by making a post to the REST interface that RackHD can then add to its data model.
 
 * Active discovery is invoked when a machine attempts to PXE boot on the network that RackHD is
-  monitoring. As a new machine PXE boots, RackHD retrieves the MAC address of the machine.
+  monitoring. As a machine PXE boots, RackHD retrieves the MAC address of the machine.
   If the MAC address has not been recorded, RackHD creates a new record in the data model and
-  then invokes a default workflow. To enable active discovery, you set the default workflow that
-  will be run when a new machine is identified to one of the discovery workflows included
-  within the system. The most common is the SKU Discovery workflow.
+  then invokes a default workflow. Usually, the SKU Discovery workflow is the default.
```

Comment:
I was trying to aim at the knowledge that "default workflow" was something configurable on the overall system, as opposed to implying that the default code functionality would do this.